src/library_signals.js

```diff
@@ -124,6 +124,19 @@ var funs = {
     ___setErrNo(ERRNO_CODES.EINTR);
     return -1;
   },
+#if ASSERTIONS
+  siglongjmp__deps: ['longjmp'],
+  siglongjmp: function(env, value) {
+    // We cannot wrap the sigsetjmp, but I hope that
+    // in most cases siglongjmp will be called later.
+
+    // siglongjmp can be called very many times, so don't flood the stderr.
+    Runtime.warnOnce("Calling longjmp() instead of siglongjmp()");
+    _longjmp(env, value);
+  },
+#else
+  siglongjmp: 'longjmp',
```

Comment:
Thank you for a clarification. In case it will be somehow fixed in a centralized manner, I hope it may be left as is.